src/main/java/com/getindata/connectors/http/internal/utils/Utils.java

```diff
@@ -0,0 +1,50 @@
+package com.getindata.connectors.http.internal.utils;
+
+import org.apache.flink.metrics.MetricGroup;
+import org.apache.flink.metrics.groups.UnregisteredMetricsGroup;
+import org.apache.flink.util.SimpleUserCodeClassLoader;
+import org.apache.flink.util.UserCodeClassLoader;
+
+public final class Utils {
```

Comment:
Shouldn't it be something more descriptive, like e.g. `SerializationSchemaUtils`? Especially considering that in the same package, we have more Utils, e.g. `HttpHeaderUtils`.

Reply:
sure, changed.